service/automatic_run_experiment.py

```diff
@@ -46,53 +46,6 @@
 
 EXPERIMENT_NAME_REGEX = re.compile(r'^\d{4}-\d{2}-\d{2}.*')
 
-# TODO(metzman): Stop hardcoding benchmarks and support marking benchmarks as
-# disabled using a config file in each benchmark.
-CODE_BENCHMARKS = [
-    # OSS-Fuzz benchmarks.
-    'bloaty_fuzz_target',
-    'curl_curl_fuzzer_http',
-    'jsoncpp_jsoncpp_fuzzer',
-    'libpcap_fuzz_both',
-    'libxslt_xpath',
-    'mbedtls_fuzz_dtlsclient',
-    'openssl_x509',
-
-    # Temporarily disabled due to out-of-memory issues with clang coverage.
-    # 'php_php-fuzz-parser',
```

Comment:
Your new change means we are going to use `php_php-fuzz-parser` again in automatic experiments.
Do we have reason to think it will work now?

Reply:
Yes it works fine for some time now
https://www.fuzzbench.com/reports/2021-02-13-paper/index.html#php_php-fuzz-parser
I think it was that num-threads=1 change that Bharat did some time back.